apps/server/src/shared/domain/entity/boardnode/richtext-element-node.entity.ts

```diff
@@ -0,0 +1,31 @@
+import { Entity, Property } from '@mikro-orm/core';
+import { AnyBoardDo } from '@shared/domain/domainobject';
+import { InputFormat } from '@shared/domain/types';
+import { BoardNode, BoardNodeProps } from './boardnode.entity';
+import { BoardDoBuilder, BoardNodeType } from './types';
+
+@Entity({ discriminatorValue: BoardNodeType.RICH_TEXT_ELEMENT })
+export class RichTextElementNode extends BoardNode {
+	@Property()
+	text: string;
+
+	@Property()
+	inputFormat: InputFormat;
+
+	constructor(props: RichTextElementNodeProps) {
+		super(props);
+		this.type = BoardNodeType.RICH_TEXT_ELEMENT;
+		this.text = props.text;
```

Comment:
I would name this prop content, to make it clear is not simple text, but is not important

Reply:
I would not name it "content" as this would be quite abstract and somehow the inputFormat should be part of such a "content"-definition. I would rather keep "text" or switch to "html".
Furthermore content is already an encapsulating part of the specialized attributes of each distinct element-type. (e.g. RichTextElementContent)

Reply:
I would like to keep 'text' then to be honest, as I also see the duplication in [here](https://github.com/hpi-schul-cloud/schulcloud-server/blob/f73091554989a9bbd0e229cde77c78dbf3fff6e9/apps/server/src/modules/board/controller/dto/element/element-content-update.body.params.ts#L15)